docs/_tooling/extensions/header_service/README.md

```diff
@@ -0,0 +1,56 @@
+# Automatic Header Generation Service
+
+## Purpose
+The service *header-service* generates a table with review and approver informations.
+It uses the template *header.need* to generate this table.
+
+## Extraction Methods
+This modules can use two different methods to extract the needed information.
+* GitHub data
+* Merge commit log
+
+The used method can be configured as a configuration parameter in *conf.py*.
+
+    header_service_use_github_data = True
+
+GitHub data is used if parameter is not set.
+
+
+## Environment variables
+This module uses the following environment variables if the extraction method **Github data** is configured:
+
+GH_TOKEN: Github access token
+GITHUB_REF_NAME: Github reference name (<pull request no>/merge)
```

Comment:
Who is setting GITHUB_REF_NAME? Shouldn't it retrieve GITHUB_REF_NAME automatically based on the current file name?

Reply:
But that's the current PR. What you want is the PR where THAT document was reviewed?!

Reply:
Oh that brings me to a new idea on how to implement this feature (if really needed, as mentioned elsewhere). We can have a script grab the info from github API and store it somewhere. So we don't need to query github every time.
Potentially triggering said script on-merge.